.github/workflows/testing-and-deployment.yml

```diff
@@ -217,6 +217,9 @@ jobs:
           pip install --pre --no-deps -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple matplotlib
           pip install --pre --no-deps -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy
 
+      - name: Test collection (temporary)
+        run: python -m pytest --collect-only
+
```

Comment:
Added temporary only to see if it fails on CI. Will revert later.

Reply:
CI has failed as expected https://github.com/pyvista/pyvista/actions/runs/12206271755/job/34055394148

Reply:
Although the failure is different from what's been observed locally ...

Reply:
Differences on CI was caused by OpenGL. Correctly fails [here](https://github.com/pyvista/pyvista/actions/runs/12213181187/job/34072507064?pr=6948#step:14:8321) .